README.md

```diff
@@ -26,89 +26,128 @@ Start by generating an Ethereum private key; this will be used as a signing key
 python3 -c "import os; import binascii; print('0x%s' % binascii.hexlify(os.urandom(32)).decode('utf-8'))"
 ```
 
-For the rest of this demo we will be using the standard test private key `0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80`.
+Please set the private key to gateway .env file under `packages/gateway/`:
+
+```bash
+cp ./packages/gateway/.env.example ./packages/gateway/.env
+code ./packages/gateway/.env
+```
 
 First, install dependencies and build all packages:
 
 ```bash
-yarn && yarn build
+yarn clean && yarn install && yarn build
 ```
 
-Next, run the gateway:
+Next, run the gateway with the private key on the `Goerli` testnet:
 
 ```bash
-yarn start:gateway --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --data test.eth.json
+yarn start-goerli:gateway --data token.eth.json
 ```
 
-The value for the `--private-key` flag should be the key you generated earlier.
+The value for the private key should be the key you set earlier in the .env file.
 
 You will see output similar to the following:
+
 ```
-Serving on port 8000 with signing address 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
+Serving on port 8000 with signing address 0x3B7D34d0E7e807A9D7aD74F094C5379aca61460D
 ```
 
-Take a look at the data in `test.eth.json` under `packages/gateway/`; it specifies addresses for the name `test.eth` and the wildcard `*.test.eth`.
+Take a look at the data in `token.eth.json` under `packages/gateway/`; it specifies addresses for the name `token.eth` and the wildcard `*.token.eth`.
+
+Next, edit `packages/contracts/scripts/goerli/deployments/AddressRecord.json`; replacing the address of `Signer` and `Owner` with the one output when you ran the command above.
 
-Next, edit `packages/contracts/hardhat.config.js`; replacing the address on line 59 with the one output when you ran the command above. Then, in a new terminal, build and run a test node with an ENS registry and the offchain resolver deployed:
+And, edit contracts .env file under `packages/contracts/`:
 
+```bash
+cp ./packages/contracts/.env.example ./packages/contracts/.env
+code ./packages/contracts/.env
 ```
-cd packages/contracts
-npx hardhat node
+
+Then, in a new terminal, connect to `Goerli` testnet with an ENS registry and the offchain resolver deployed:
+
+```bash
+cd ./packages/contracts
+npx hardhat run ./scripts/deploy/ENSRegistry.ts --network goerli
+npx hardhat run ./scripts/deploy/OffchainResolver.ts --network goerli
 ```
 
-You will see output similar to the following:
+Before setting the domain owner, move the new `ABI` files to `deployments` folder:
 
+```bash
+mkdir -p ./deployments/goerli
+mv -f ./scripts/goerli/deployments/ENSRegistry.json ./deployments/goerli/ENSRegistry.json
```

Comment:
I think we can do it in next PR to write the deployment info directly to `./deployments` so others can skip this step?

Reply:
I think because on the "Hardhat" network executing a simulated deployment contract in Private Repo will permanently overwrite the abi files in "Mainnet" folder,
it will be manually overwritten by the user.

Not sure if it's safer to keep this traditional way in our offchain-resolver Repo?